docs/testrunner-toolkit/configuration/puppeteer.md

```diff
@@ -385,6 +385,17 @@ One or more paths to the puppeteer test files to run for this suite. Regex value
     testMatch: ["**/*.js"]
 ```
 
+---
+
+### `browserArgs`
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+Pass browser args as the launch options of puppeteer.
```

Comment:
```suggestion
Pass flags to configure how Puppeteer launches the selected browser. Review supported flags for [Chrome/Chromium](https://peter.sh/experiments/chromium-command-line-switches/) or [Firefox](The link from Puppeteer is broken).
```
@tianfeng92  -- am I understanding this property correctly? I looked at this page: https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-puppeteerlaunchoptions

Reply:
I can't really find anything for Firefox that provides a list of valid arguments. The closest page I found was here: https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/firefoxOptions, which doesn't really provide a "list" of potential values. Maybe we just provide the Chrome link?

Reply:
I think it would be enough to only provide the chrome link currently 🤔

Reply:
Let me search that a bit. I don't think https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-puppeteerlaunchoptions is providing `browser args`. It's just launching options of puppeteer.

Reply:
found one for firefox http://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options.html
But it's from third-party web archive provider. Is it fine to put that in our docs?

Reply:
Oh...sorry. I was reviewing the wrong commit. I would ask @KunalJain if it is fine to include that link.